Configuration.mk

```diff
@@ -286,28 +277,37 @@ override WLFLAGS_rv32imac += $(WLFLAGS_rv32)
 
 # Set the system libraries we link against for RISC-V. We support C++ apps by
 # default.
-override LINK_LIBS_rv32 += \
-      -lgcc -lstdc++ -lsupc++
+ifeq ($(PICOLIBC),)
+      override LINK_LIBS_rv32 += \
```

Comment:
Can you update the indentation to match the other ifeq (i.e. two spaces)?

Reply:
Fixed!